botocore/utils.py

```diff
@@ -2983,22 +2997,20 @@ def _get_response(self, full_url, headers, timeout):
             if response.status_code != 200:
                 raise MetadataRetrievalError(
                     error_msg=(
-                        "Received non 200 response (%s) from ECS metadata: %s"
+                        f"Received non 200 response {response.status_code} "
+                        f"from container metadata: {response_text}"
                     )
-                    % (response.status_code, response_text)
                 )
             try:
                 return json.loads(response_text)
             except ValueError:
-                error_msg = (
-                    "Unable to parse JSON returned from ECS metadata services"
-                )
-                logger.debug('%s:%s', error_msg, response_text)
+                error_msg = "Unable to parse JSON returned from container metadata services"
+                logger.debug(f"{error_msg}: {response_text}")
```

Comment:
This is what I was noting in the previous comment from https://github.com/boto/botocore/pull/3039#discussion_r1372386119. We can leave log statements alone because they're actually more performant with the older syntax. This is mostly for error messages we're returning in exceptions.